src/components/AddNewProject/AddNewProject.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import { useTranslation } from "react-i18next";
+// import { Link } from "react-router-dom";
+// import { useNavigate } from 'react-router-dom';
+// import { toast } from "react-toastify";
```

Comment:
do usunięcia?

Reply:
zostawiłam sobie to na później, gdy będę robić połączenie z bazą. nie chciałam tak wielkiej pr wystawiać :P